modules/gapi/include/opencv2/gapi/cpu/gcpukernel.hpp

```diff
@@ -210,12 +210,10 @@ struct tracked_cv_mat{
 
 struct scalar_wrapper
 {
-    scalar_wrapper(cv::gapi::own::Scalar& s) : m_s{cv::gapi::own::to_ocv(s)}, m_org_s(s) {};
-    operator cv::Scalar& () { return m_s; }
-    void writeBack() const  { m_org_s = to_own(m_s); }
+    scalar_wrapper(cv::Scalar& s) : m_org_s(s) {};
+    operator cv::Scalar& () { return m_org_s; }
 
-    cv::Scalar m_s;
-    cv::gapi::own::Scalar& m_org_s;
+    cv::Scalar& m_org_s;
 };
```

Comment:
This whole thing may be not necessary now.

Previously it has served as an adaptor between kernels (which continued using `cv::Scalar`) and G-API internals (which were `cv::gapi::own::Scalar`-based)

Reply:
Yes. Removed scalar_wrapper struct. Fixed.